.github/workflows/pull-build.yml

```diff
@@ -0,0 +1,47 @@
+name: Build and test library
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: windows-latest
+
+    steps:
+    - run: mkdir -p artifactApp
+    - run: mkdir -p artifactTests
+
+    - uses: actions/checkout@master
+
+    - uses: actions/checkout@v2
```

Comment:
Is it intentional/required to have 2 checkout actions?

Reply:
No it was not